tests/osgi-tests/src/test/java/org/wso2/carbon/datasource/osgi/DataSourceServiceTest.java

```diff
@@ -34,6 +34,9 @@
 
 import static org.ops4j.pax.exam.CoreOptions.mavenBundle;
 
+/**
+ * Test class for {@code DataSourceManagementService}.
```

Comment:
Same as above, the better approach might be to use @link:
http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#link